pkg/controlplane/controller/defaultservicecidr/default_servicecidr_controller.go

```diff
@@ -199,5 +217,8 @@ func (c *Controller) syncStatus(serviceCIDR *networkingapiv1.ServiceCIDR) {
 			klog.Infof("error updating default ServiceCIDR status: %v", errApply)
 			c.eventRecorder.Eventf(serviceCIDR, v1.EventTypeWarning, "KubernetesDefaultServiceCIDRError", "The default ServiceCIDR Status can not be set to Ready=True")
 		}
+	} else {
```

Comment:
Trying to get oriented to what `syncStatus` was doing previously... it looks like if it has a ServiceCIDRConditionReady condition with any value, `syncStatus` short-circuits and will never reach here... is that right?

It seems like in an upgrade scenario where this ServiceCIDR already existed and was ready, that means we'll never see this log message or event?

If we *did* get in a situation where we saw this, this would make the API server log and report this event every 10 seconds?

Reply:
you are right, it should only log once